src/cdk/a11y/a11y-module.ts

```diff
@@ -6,7 +6,8 @@
  * found in the LICENSE file at https://angular.dev/license
  */
 
-import {ObserversModule} from '@angular/cdk/observers';
+// @ng_package: ignore-cross-repo-import
+import {ObserversModule} from '../observers';
```

Comment:
I was under the impression that cross-package imports should still be absolute?

Reply:
I checked with @devversion to confirm that everything within `src/material` should be relative to each other since they are all shared under the `@angular/material` npm package. Likewise with cdk